lib/components/form/advanced-settings-panel.tsx

```diff
@@ -213,11 +223,12 @@ const AdvancedSettingsPanel = ({
         fillModeIcons
         label="test"
         modeButtons={processedModeButtons}
-        onSettingsUpdate={onSettingsUpdate(setQueryParam)}
-        onToggleModeButton={setModeButton(
-          enabledModeButtons,
-          onSettingsUpdate(setQueryParam)
+        onSettingsUpdate={onAdvancedModeSubsettingsUpdate(
```

Comment:
What I am uncomfortable with here is that the advanced settings panel already has the mode button that corresponds with the mode setting that is being updated. I don't think we should do a bunch of work to get that mode button item again. It will require an update to the OTP UI component but I think there's a better way to go about this.

Reply:
In fact, maybe it would be possible to do this change almost entirely within OTP UI? Could we just add a prop for `transportModesDisableCascade`? Weird name idk but something like that